lib/ssl/src/tls_v1.erl

```diff
@@ -203,30 +207,29 @@ finished(Role, Version, PrfAlgo, MasterSecret, Handshake)
 
 %% TODO 1.3 finished
 
--spec certificate_verify(md5sha | sha, integer(), [binary()]) -> binary().
-
+-spec certificate_verify(HashAlgo, Handshake) -> binary() when
+      HashAlgo :: md5sha | sha,
```

Comment:
should have type ssl:hash() even though ignored by the second clause for optimization reasons.